apps/web/pages/booking/[uid].tsx

```diff
@@ -165,10 +165,13 @@ export default function Success(props: SuccessProps) {
     );
   }
 
+  const evtName = !eventType.isDynamic
+    ? (props.dynamicEventName as string) || props.eventType.eventName
+    : bookingInfo.responses?.title;
   const eventNameObject = {
     attendeeName,
     eventType: props.eventType.title,
-    eventName: (props.dynamicEventName as string) || props.eventType.eventName,
+    eventName: evtName,
     host: props.profile.name || "Nameless",
     location: location,
     bookingFields: bookingInfo.responses,
```

Comment:
Perhaps a good idea to filter `bookingFields` and remove title to avoid duplication of the same information in two places. You're probably going to need to filter this at the point of rendering as well.

![image](https://github.com/calcom/cal.com/assets/52925846/0fe07161-666c-4b84-9f3d-b07314f54a22)